xclim/indices/fire/_ffdi.py

```diff
@@ -239,10 +239,20 @@ def keetch_byram_drought_index(
     """
 
     def _keetch_byram_drought_index_pass(pr, tasmax, pr_annual, kbdi0):
-        """Pass inputs on to guvectorized function `_keetch_byram_drought_index`. DO NOT CALL DIRECTLY, use `keetch_byram_drought_index` instead."""
-        # This function is actually only required as xr.apply_ufunc will not receive
-        # a guvectorized function which has the output(s) in its function signature
-        return _keetch_byram_drought_index(pr, tasmax, pr_annual, kbdi0)
+        """Pass inputs on to guvectorized function `_keetch_byram_drought_index`.
+
+        This function is actually only required as `xr.apply_ufunc` will not receive
+        a guvectorized function which has the output(s) in its function signature.
+
+        Warnings
+        --------
+        DO NOT CALL DIRECTLY, use `keetch_byram_drought_index` instead.
+        """
+        # fmt: off
+        return _keetch_byram_drought_index(  # pylint: disable=no-value-for-parameter
+            pr, tasmax, pr_annual, kbdi0
+        )
+        # fmt: on
```

Comment:
Hmm, I understand what's going on here, but I find it a bit verbose to have three comments  only to disable a warning. A warning that kinda is an error from pylint in fact.

Reply:
That's fair. Resolved.